stories/grid/nested.js

```diff
@@ -7,6 +7,7 @@ import Grid from '../../src/grid'
 import { times } from '../../src/utils'
 import Box from '../box'
 import { getBoxType } from '../utils'
+import styles from '../../src/flex.css'
```

Comment:
I'm wondering if it's worth adding a linter rule to enforce all named css imports set the variable name to 'styles'